features/providers/step_definitions/civil_journey_steps.rb

```diff
@@ -346,8 +346,12 @@
   expect(page).to have_content("Enter your client's home address")
 end
 
-Then('I am on the benefit check results page') do
-  expect(page).to have_content("Your client's tax and benefits status")
+Then('I am on the benefit check results page and the benefits qualify') do
+  expect(page).to have_content('receives benefits that qualify for legal aid')
+end
```

Comment:
Rather than create page specific step definitions there is already a step definition that would more clearly expresses what is happening here: "Then I should be on a page showing"

So remove these two step definitions, and in the features when you want to check the citizen received benefits use:
```
Then I should be on a page showing "receives benefits that qualify for legal aid"
```
And when the benefits don't qualify:
```
Then I should be on a page showing "must complete a financial assessment"
```

Reply:
I'm not quite following this I'm afraid.  
I changed it like this as there was no common text between the two pages now, aside from something too generic like "what happens next".  This seemed akin to removing the test completely, so we opted for this approach, splitting up the two distinct pages.